spec/models/video_spec.rb

```diff
@@ -1,24 +1,7 @@
 require 'spec_helper'
 
 describe Video do
-  it "saves itself" do
-    video = Video.new(title: "Superman", description: "Man of Steel")
-    video.save
-    expect(Video.first).to eq(video)
-  end
-
-  it "belongs to category" do
-    dramas = Category.create(name: "dramas")
-    monk = Video.create(title: "Monk", description: "A drama", category: dramas)
-    expect(monk.category).to eq(dramas)
-  end
-
-  it "does not save a video without a title" do
-    video = Video.create(description: "A good video!")
-    expect(Video.count).to eq(0)
-  end
-
-  it "does not save a video without a description" do
-    video = Video.create(title: "Monk")
-    expect(Video.count).to eq(0)
-  end
\ No newline at end of file
+  it { should belong_to(:category)}
```

Comment:
For the correct ruby style, you want to keep spaces around both brackets. `it { should belong_to(:category) }` - works same for below specs.

Note: it also works with no spaces well, but it should be the same on both sides.